### PR TITLE
Feature/127/isempty

### PR DESCRIPTION
**Pull request by MNie**
#127 
CC: @haf

### CODE REVIEW

File: Expecto/Expect.fs [hunk 1]
```diff
@@ -170,6 +170,19 @@ let isNotWhitespace (actual : string) message =
...
+/// Expect the passed sequence to be empty.
+let isEmpty (actual) message =
+  if not (Seq.isEmpty actual) then Tests.failtestf "%s. Should be empty." message
+
+/// Expect the passed sequence to be not empty.
+let isNonEmpty (actual) message =
+  if Seq.isEmpty actual then Tests.failtestf "%s. Should be empty." message
+
+/// Expect that the counts of the found value occurrences in sequence equal the expected.
+let hasCountOf (actual : _ seq) (expected : uint32) (selector : _ -> bool) message =
```
Comment (haf) [inline Expecto/Expect.fs]:
```
    let len =
      xs |> Seq.fold (fun acc x -> if selector x then acc + 1u else acc) 0u
    Expect. ...
```

Would probably be more efficient.

**Comment by haf** [comment]
Nice!

File: Expecto/Expect.fs (hunk 1 above)
Comment (MNie) [inline Expecto/Expect.fs]: Fix'd :)
CC: @haf

**Comment by haf** [comment]
Sorry, I forgot; do you think you could add these to the README as well?

**Comment by MNie** [comment]
@haf but I did that [in this commit](https://github.com/haf/expecto/pull/128/commits/860f3d06447a3b39e3f213ce87c8916adff2a4b6) :)

**Comment by haf** [comment]
True, don't mind me ;)

**Pull request merged**